src/workerd/api/actor.h

```diff
@@ -73,7 +73,8 @@ class DurableObject: public Fetcher {
 public:
   DurableObject(jsg::Ref<DurableObjectId> id, IoOwn<OutgoingFactory> outgoingFactory,
                 RequiresHostAndProtocol requiresHost)
-    : Fetcher(kj::mv(outgoingFactory), requiresHost, true /* isInHouse */),
+    : Fetcher(kj::mv(outgoingFactory), requiresHost, true /* isInHouse */,
+          false /* supportsConnect */),
```

Comment:
boolean args are evil. 

I know it's a bit of a yak shaving exercise but can we change this into an options struct like...

```
struct Fetcher::Options {
  bool inHouse = false;
  bool supportsConnect = false;
};
```

Or use an enum 

```
enum FetcherOptions {
  NONE,
  IN_HOUSE,
  SUPPORTS_CONNECT
};
```